docs/src/main/paradox/examples/ftp-samples.md

```diff
@@ -40,7 +41,7 @@ Scala
 
```

Comment:
The snippet directive should go after the Scala one above:

```
Scala
: @@snip [snip](/doc-examples/src/main/scala/ftpsamples/RotateLogsToFtp.scala) { #sample }

Java
: @@snip [snip](/doc-examples/src/main/java/ftpsamples/RotateLogsToFtpExample.java) { #sample }
```

You can also see how the documentation looks like after your changes by running `docs/previewSite` in sbt.

Reply:
:O sorry about that!

The following could also be a good idea?
```
Java
:   ```
    sbt
    > doc-examples/runMain ftpsamples.FtpToFileExample
    > doc-examples/runMain ftpsamples.RotateLogsToFtpExample
    ```
```

BTW site generation seems to be broken because of `GoogleTokenApi`.

Reply:
Yes, the run example looks good. Can you post the output of the `doc/previewSite` when it fails to generate the docs?

Reply:
this is the error reported by SBT

```
[error] /Users/me/Documents/workspace/alpakka/google-cloud-pub-sub/src/main/scala/akka/stream/alpakka/googlecloud/pubsub/impl/GoogleTokenApi.scala:26:27: could not find implicit value for parameter clock: java.time.Clock
[error]   def now: Long = JwtTime.nowSeconds
[error]                           ^
[error] /Users/me/Documents/workspace/alpakka/google-cloud-pub-sub/src/main/scala/akka/stream/alpakka/googlecloud/pubsub/impl/GoogleTokenApi.scala:30:17: could not find implicit value for parameter clock: java.time.Clock
[error]       .expiresIn(3600)
[error]                 ^
...
[error] /Users/me/Documents/workspace/alpakka/google-fcm/src/main/scala/akka/stream/alpakka/google/firebase/fcm/impl/GoogleTokenApi.scala:29:27: could not find implicit value for parameter clock: java.time.Clock
[error]   def now: Long = JwtTime.nowSeconds
[error]                           ^
[error] /Users/me/Documents/workspace/alpakka/google-fcm/src/main/scala/akka/stream/alpakka/google/firebase/fcm/impl/GoogleTokenApi.scala:33:17: could not find implicit value for parameter clock: java.time.Clock
[error]       .expiresIn(3600)
[error]                 ^
```

Reply:
That is strange. I remember that from some time ago where a dependency in google-fcm connector changed. But current master should be fine.